### PR TITLE
Maven project

### PR DESCRIPTION
**Pull request by AlexanderVladimirov9090**
Done With Maven.

### CODE REVIEW

File: .idea/buildtools.iml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (EmilGeorgiev) [inline .idea/buildtools.iml]: folder  ```.idea``` should be in .gitignore

File: myproject-bo/src/test/java/com/clouway/myprojectbo/MainTest.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+/**
+ * Created by clouway on 14.07.16.
+ *
+ * @author Alexander Vladimirov
+ *         (alexandervladimirov1902@gmail.com)
+ */
+public class MainTest {
+
+}
```
Comment (EmilGeorgiev) [inline myproject-bo/src/test/java/com/clouway/myprojectbo/MainTest.java]: Why missing implementation of this class.

File: .idea/buildtools.iml (hunk 1 above)
Comment (AlexanderVladimirov9090) [inline .idea/buildtools.iml]: yeah will be done.

File: myproject-bo/src/test/java/com/clouway/myprojectbo/MainTest.java (hunk 2 above)
Comment (AlexanderVladimirov9090) [inline myproject-bo/src/test/java/com/clouway/myprojectbo/MainTest.java]: Will do it.

File: myproject-pl/pom.xml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>jar</packaging>
+  <name>myproject-pl</name>
```
Comment (EmilGeorgiev) [inline myproject-pl/pom.xml]: this project should have dependency to jdbs driver of MySQL

**Comment by EmilGeorgiev** [comment]
I can't see jarjar plugin. We should use it to include dependencies in jar file

**Comment by AlexanderVladimirov9090** [comment]
Will be done.

**Comment by AlexanderVladimirov9090** [comment]
Forgot to commit and push that is why jarjar Plugin was missing.

File: myproject-pl/pom.xml (hunk 3 above)
Comment (AlexanderVladimirov9090) [inline myproject-pl/pom.xml]: Know, delete it by accident will be added.

**Comment by AlexanderVladimirov9090** [comment]
Done.

**Comment by EmilGeorgiev** [comment]
LGTM

**Pull request closed without merging**